### ISSUE TITLE
Chaos Space Marines - Terminator heavy weapon selection limit issue

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#3218]
**File:** Chaos Space Marines - Codex.catz

**BattleScribe version:** 2.00.22

**Platform:** Android

**Dropbox:** No

**Description:** I found a flawless in your codex chaos space marine roster about the chaos terminators. If you have a champion, 7 terminators, and 2 heavy wepon terminators, according to the actual codex, you may take 1 heavy wepon for 5 models or 2 for 10 models. But battle scribes list always shows an error that of you have 10 models and 2 heavy wepons, that you have 1 to many heavy wepons. 
Normally I would understand if maybe it was a 20 man unit and you got a heavy wepon for every 10. But this is not the case, for the maximum for a terminator squad is only 10 models. 
Is thier a way to fix this?

**Comment by cartag** [comment]
I set it up to check if there were at least five selections, and set it to 1, and check for 10 and set to 2. It either checked the first and ignored the second, or fired on the second and then fired on the first. Either way it ended at 1. 

Tldr; Put limits on your stuff.
